verticapy/performance/vertica/collection/collection_tables.py

```diff
@@ -54,7 +57,61 @@ class BundleVersion(Enum):
     """
 
     V1 = 1
-    LATEST = V1
```

Comment:
what's that?

Reply:
Updated class docstring and added comments to each version